docs/index.md

```diff
@@ -18,7 +18,7 @@ After inference completes, MIG receives notifications for exporting the results
 
 ## Services
 
-*MONAI Deploy Informatics Gateway* contains the following standard protocols to communicate with your medical devices:
+MIG contains the following standard protocols to communicate with medical devices:
 
 * **DICOM SCP**: C-ECHO, C-STORE
 * **DICOM SCU**: C-STORE
```

Comment:
Are the  DICOMweb client and FHIR client protocols subsets of the ACR DSI API? The indenting would suggest so.

Reply:
They are not, but the only way of using them is by calling the ACR API.